catalog/build.gradle

```diff
@@ -51,6 +51,7 @@ def srcDirs = [
 
 android {
   defaultConfig {
+    vectorDrawables.useSupportLibrary = true
```

Comment:
This isn't necessary. I think the default might be true now?

Reply:
I'm not sure! cause if you don't set this flag, will be generated png files for vectors if your minSdk lower 5.0